STUDIO/org.openl.rules.webstudio/webapp/pages/modules/repository/topPanel.xhtml

```diff
@@ -28,7 +28,7 @@
 
         <div class="navbar-inner">
             <h:outputLink onclick="openNewProjectDialog();return false;" styleClass="primary"
-                title="Create a new project" rendered="#{repositoryTreeState.canCreate}">
+                title="Create a new project" rendered="#{repositoryTreeState.canCreate and (repositoryTreeController.canCreateNewProject or not empty repositoryTreeController.nonFlatRepositories)}">

```

Comment:
What about **Import From Repository** tab (if we have any non-flat git repository)?
Even if we can't create a new project in any repository, we still can import existing project into WebStudio.
Can we improve the condition for "Create Project" button to take this into account?